### DISCUSSION TITLE
Verbindung Lokal

### DISCUSSION THREAD

**Discussion by NokmanAgo** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#11] [General]
Mahlzeit, 
gibt es einen Grund warum das nur entweder lokal oder über die cloud geht?

**Comment by KaLaeu** [comment]
Schreckt mich auch aktuell ab. Hätte gern beides parallel. Da war Version 1 besser.

**Comment by mawoka-myblock** [comment]
Das müsste @SonnenladenGmbH mal bei APsystems hinterfragen.
